pyaedt/modules/report_templates.py

```diff
@@ -375,93 +567,6 @@ def add_cartesian_y_marker(self, val, name=None, y_axis=1):  # pragma: no cover
             return name
         return ""
 
-
-class Standard(CommonReport):
-    """Standard Report Class that fits most of the application standard reports."""
```

Comment:
Provides a reporting class that fits most of the application's standard reports.